### PR TITLE
LPS-149354 - fix Zip Slip vulnerability in ContainerLPKGUtil.deploy

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

Try to simplify the unit test based on https://github.com/liferay-core-infra/liferay-portal/pull/1494.
This is for ticket https://issues.liferay.com/browse/LPS-149354.
Thanks for checking!

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@jeffwu0724 this [failure](https://github.com/liferay-core-infra/liferay-portal/pull/1494#issuecomment-1231907039) is still happening, can you take a look? thanks!

File: modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/test/java/com/liferay/portal/lpkg/deployer/internal/ContainerLPKGUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+			Assert.assertEquals(logEntries.toString(), 1, logEntries.size());
+
+			LogEntry logEntry = logEntries.get(0);
+
+			Assert.assertEquals(
+				"Invalid LPKG File name: ../bad.lpkg", logEntry.getMessage());
+		}
+	}
+
+	private List<File> _createExpectedOutputFileList(String fileName) {
```
Comment (tinatian) [inline modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/test/java/com/liferay/portal/lpkg/deployer/internal/ContainerLPKGUtilTest.java:122]: @jeffwu0724 Please inline this method. It is only one line, do not need to be a method.

**Comment by jeffwu0724** [comment]
Also double checked locally that the possible regression mentioned [here](https://github.com/liferay-core-infra/liferay-portal/pull/1494#issuecomment-1231907039) cannot be reproduced

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
Closed, as we talked, please resend.

Thanks.
Tina

**Comment by catarineaguiar** [comment]
ci:stop

**Pull request closed without merging**